db/range_del_aggregator.cc

```diff
@@ -168,7 +193,8 @@ class CollapsedRangeDelMap : public RangeDelMap {
   const Comparator* ucmp_;
 
  public:
-  CollapsedRangeDelMap(const Comparator* ucmp) : ucmp_(ucmp) {
+  CollapsedRangeDelMap(const Comparator* ucmp)
+      : rep_(stl_wrappers::LessOfComparator(ucmp)), ucmp_(ucmp) {
```

Comment:
@ajkr, @benesch discovered this when I added `IteratorRangeTombstoneCoversSstableEndpointsCustomComparator`. `LessOfComparator` defaults to using `BytewiseComparator`.

Reply:
Heh, I guess I should have pulled this out into a separate PR.